Kernel/Devices/GPU/DisplayConnector.h

```diff
@@ -169,6 +173,10 @@ class DisplayConnector : public CharacterDevice {
     LockWeakPtr<Process> m_responsible_process;
     Spinlock<LockRank::None> m_responsible_process_lock {};
 
-    IntrusiveListNode<DisplayConnector, LockRefPtr<DisplayConnector>> m_list_node;
+    mutable IntrusiveListNode<DisplayConnector> m_list_node;
+
+public:
+    using List = IntrusiveList<&DisplayConnector::m_list_node>;
+    static SpinlockProtected<DisplayConnector::List, LockRank::None>& all_instances();
```

Comment:
Do we not want to put this at the top of the list? At least the presence of `all_instances` would somewhat match the other static methods.